src/open_inwoner/configurations/bootstrap/utils.py

```diff
@@ -0,0 +1,82 @@
+from dataclasses import dataclass
+from typing import Any, Iterator
+
+from django.db.models.fields import NOT_PROVIDED
+
+
+@dataclass(frozen=True)
+class ConfigField:
+    name: str
+    verbose_name: str
+    description: str
+    field_type: str
+    default_value: str
+    values: str
+
+
+def populate_fields(
+    fields: dict[str, list],
+    require: tuple[str],
+    exclude: tuple[str],
+    all_fields: Iterator,
+):
+    for field in all_fields:
+        config_field = ConfigField(
+            name=field.name,
+            verbose_name=field.verbose_name,
+            description=field.description,
+            field_type=field.get_internal_type(),
+            default_value=get_default_value(field),
+            values=get_example_value(field),
+        )
+        fields["all"].append(config_field)
+        if require and config_field.name in require:
+            fields["required"].append(config_field)
```

Comment:
In general let's try to move away from ad-hoc dicts with important keys and use dataclasses.